recipes-extended/xen/xen-tools_git.bbappend

```diff
@@ -14,6 +14,8 @@ B = "${S}"
 
 DEFAULT_PREFERENCE = "1"
 
+EXTRA_OECONF+="--disable-golang"
```

Comment:
What's the motivation for this change?

Reply:
I forget - I'll investigate and try enabling golang.  I wouldn't have added this without a reason, so there may have been a build issue.  The flipside is since OpenXT doesn't use Go, I may have disabled it to cut down on build time.

Reply:
In my test, configure didn't find "go" and successfully compiled.  I will remove this.

Reply:
Interesting, and thanks.  We've been using golang downstream and could see upstreaming some of that work; not sure if it matters for xen or any of the xen tools specifically though.